HelpSource/Classes/Nil.schelp

```diff
@@ -7,6 +7,16 @@ description::
 Nil has a single instance named nil and is used to represent uninitialized data,
 bad values, or terminal values such as end-of-stream.
 
+note::
+As Nil has a single instance, comparing it against code::nil:: will not produce the results you expect: Nil can not have a value of code::nil::. In normal use you should not need to use this class directly: use code::nil::
```

Comment:
this should probably just say 'Nil is a class, comparing it against..'?

Reply:
Yes, I think `Nil is a class` would be good.

Reply:
i would phrase this as "Take note that Nil is a class, and not the special value code::nil::. Comparing Nil and code::nil:: will return false. In normal use [...]"